src/snackbar/index.tsx

```diff
@@ -1,7 +1,6 @@
-import { WidgetBase } from '@dojo/framework/core/WidgetBase';
-import { DNode, RenderResult } from '@dojo/framework/core/interfaces';
-import { theme, ThemedMixin } from '@dojo/framework/core/mixins/Themed';
-import { tsx } from '@dojo/framework/core/vdom';
+import { RenderResult } from '@dojo/framework/core/interfaces';
+import { create, tsx } from '@dojo/framework/core/vdom';
+import theme from '@dojo/framework/core/middleware/theme';
 import * as css from '../theme/snackbar.m.css';
 
 export interface SnackbarProperties {
```

Comment:
@agubler is it good for us to extend `ThemedProperties` for handling custom elements?

Reply:
You're probably right. I extended `ThemedProperties`.